src/main/scala/uk/gov/nationalarchives/Lambda.scala

```diff
@@ -20,25 +22,33 @@ import scala.xml.PrettyPrinter
 class Lambda extends RequestStreamHandler {
   val dAS3Client: DAS3Client[IO] = DAS3Client[IO]()
   implicit val inputReader: Reader[StepFnInput] = macroR[StepFnInput]
+  implicit val loggerName: LoggerName = LoggerName("Ingest Parent Folder Opex Creator")
+  private val logger: SelfAwareStructuredLogger[IO] = Slf4jFactory.create[IO].getLogger
 
   def handleRequest(input: InputStream, output: OutputStream, context: Context): Unit = {
     val rawInput: String = Source.fromInputStream(input).mkString
     val stepFnInput = default.read[StepFnInput](rawInput)
     val keyPrefix = s"opex/${stepFnInput.executionId}/"
     val opexFileName = s"$keyPrefix${stepFnInput.executionId}.opex"
-
+    val batchRef = stepFnInput.executionId.split("-").take(3).mkString("-")
+    val log = logger.info(Map("batchRef" -> batchRef))(_)
```

Comment:
`batchRef` or `batchId` to match our ddb attribute and idx name?

Reply:
`batchRef` is fine. As I said, this is only for whoever is doing support to track the progress so it's not a major thing.